.editorconfig

```diff
@@ -0,0 +1,207 @@
+# Remove the line below if you want to inherit .editorconfig settings from higher directories
+root = true
+
+# C# files
+[*.cs]
+
+#### Core EditorConfig Options ####
+
+# Indentation and spacing
+indent_size = 4
+indent_style = space
+tab_width = 4
+trim_trailing_whitespace = true
+
+# New line preferences
+end_of_line = crlf
```

Comment:
This might be a bad interaction with git, but your latest commits all seem to end with LF, not CRLF. Maybe we should change this pref to `lf` to avoid bad interactions with git? (This is just a theory I did not validate - I might be wrong)

Reply:
I checked locally, all files `*.cs` are set as `CRLF` - that's matching `.editorconfig` so it should work whatever we set here
Logically, C# devs should work from Win platform mostly, so I would leave like this `end_of_line = crlf`

Reply:
Hmm, the problem is that the CI runs on Linux and thus fails. Also there are devs working on macOS (like me) or Linux that then get all files reformatted when running `dotnet format`. I think we should get rid of the `end_of_line` directive to avoid those issues, as AFAIU git will checkout files with the platform's native line endings anyway.